src/data/profiles.js

```diff
@@ -144,6 +144,71 @@ const officers = [
 		programming, I like gaming, playing bass, watching anime and rewatching Gilmore Girls,
```

Comment:
Can you also delete Nareh's bio and photos and move them to the `alumni.js` file? You can put them under year 2024.